openshift_metrics/utils.py

```diff
@@ -132,7 +132,7 @@ def write_metrics_log(metrics_dict, file_name, openshift_cluster_name):
             req_tres = 'cpu=%s,mem=%s' % (req_cpu, req_mem)
             alloc_tres = req_tres
             info_list = [
-                str(job_id), str(job_id), cluster_name, partition_name, qos_name, account_name, group_name, str(gid_number),
+                str(job_id), str(job_id), cluster_name, partition_name, qos_name, account_name, namespace, str(gid_number),
                 user_name, str(uid_number), submission_time, eligible_time, start_time, end_time, duration, '', status, str(node_count),
                 str(cpu), str(req_cpu), str(req_mem), req_tres, alloc_tres,
                 duration, '', job_name
```

Comment:
Instead of changing it here, I'd suggest setting `group_name = namespace` above. The variable names here are intended to correspond to the xdmod database columns.

Reply:
Done, thanks!